style.css

```diff
@@ -113,3 +136,222 @@ body {
   height: 20px;
   width: 14.7px;
 }
+
+.mobile-version {
+  display: grid;
+}
+
+.second-phase {
+  margin: 114px 24px 0;
+}
+
+.second-page {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px 16px 28px;
+  display: flexbox;
+}
+
+.bar-word {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+  margin-top: 12px;
+}
+
+.nav-list {
+  margin: 20px 12px 20px 0;
+}
+
+.nav-items {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  position: relative;
+  left: -3.5rem;
+}
+
+.canopy {
+  list-style-type: none;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  color: #344563;
+  margin-right: auto;
+  margin-left: 16px;
+  line-height: 16px;
+}
+
+.back-end {
+  position: relative;
+  right: 0.6rem;
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.year {
+  position: relative;
+  right: -1.3rem;
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.small-text-2 {
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  height: 295px;
+  width: 295px;
+}
+
+.language {
+  display: flex;
+  align-items: flex-start;
+  list-style-type: none;
+  color: #6070ff;
+  position: relative;
+  left: -2.5rem;
+  margin-top: 12px;
+}
+
+.language-type {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.project-button {
+  color: #396df2;
```

Comment:
- [ ] I kindly suggest that you add the correct button interaction for enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the enabled state has the `color: #6070FF;` not `color: #396df2`

### EXAMPLE:
**for enable(default button) state**
```
button {
   color: #6070FF;
}
```